tsconfig.json

```diff
@@ -28,6 +28,7 @@
     "noUncheckedIndexedAccess": true,
     "outDir": "./dist",
     "declaration": true,
-    "removeComments": true
+    "removeComments": true,
+    "skipLibCheck": true
```

Comment:
cf. https://vitejs.dev/guide/migration.html#rollup-4

"If you are using TypeScript, make sure to set moduleResolution: 'bundler' (or node16/nodenext) as Rollup 4 requires it. Or you can set skipLibCheck: true instead."